doc/api/cluster.md

```diff
@@ -859,6 +868,59 @@ socket.on('data', (id) => {
 });
 ```
 
+## Custom Scheduling
+<!-- YAML
+added: REPLACEME
+-->
+
+The cluster master's scheduling algorithm can be customized by setting
+`cluster.schedulingPolicy` to `cluster.SCHED_CUSTOM` and passing a function
+as the `scheduler` option to `cluster.setupMaster()`. The `scheduler` is a
```

Comment:
I could make that change. In that case, does it even make sense to keep this constant?